README.markdown

```diff
@@ -1,4 +1,8 @@
-JSZip
+JSZip [![Build Status](https://api.travis-ci.org/Stuk/jszip.svg?branch=master)](http://travis-ci.org/Stuk/jszip) [![Code Climate](https://codeclimate.com/github/Stuk/jszip/badges/gpa.svg)](https://codeclimate.com/github/Stuk/jszip)
+=====
+
+[![Selenium Test Status](https://saucelabs.com/browser-matrix/jszip.svg)](https://saucelabs.com/u/jszip)
+
 =====
```

Comment:
This line was here to format "JSZip" as a title but since you added an other one above, you should remove it (it generates an empty title line, see [here](https://github.com/evilaliv3/jszip/tree/update-browser-testing#readme)).